module_cheatsheets/Machine_Learning_Cheatsheet.md

```diff
@@ -0,0 +1,86 @@
+# Machine Learning Cheatsheet
+
+#### The tables below consist of valuable functions or commands that will help you through this module. 
+##### Each table represents a different library/tool and its corresponding commands. 
+> Please note that these tables are not intended to tell you all the information you need to know about each command. 
+> 
+> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command. 
+ 
+<br>
+<br>
+
+### AnnotationDbi 
+Read the `AnnotationDbi` package vignette [**here**](http://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf).
+
+| Library/Package                                  | Piece of Code                               | What it's called                               | What it does                                                                                                                                   |
+|-----------------------------------------|---------------------------------------------|-------------------------|------------------------------------------------------------------------|
+| `AnnotationDbi`        | [`keytypes()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects)      | Keytypes     | Returns a character vector of column names/keytypes (eg., type of gene identifiers, which was "ENSEMBL" in the notebook)                                              |
```

Comment:
```suggestion
| `AnnotationDbi`        | [`keytypes()`](https://www.rdocumentation.org/packages/AnnotationDbi/versions/1.44.0/topics/AnnotationDb-objects)      | Keytypes     | Returns a character vector of column names/keytypes (eg., type of gene identifiers) available in an `AnnotationDbi` package.                                           |
```

Reply:
Okay, looks good. I assume after I make this change I'll be ready to merge, correct?

Reply:
Yep! Provided everything else has been addressed at this point (didn't look at that, trusting you).